pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/db/TransactionWrapper.java

```diff
@@ -446,6 +431,9 @@ private HikariDataSource createDataSource(int maxConnections, String dbUrl, DB d
         config.setMaxLifetime(TimeUnit.SECONDS.toMillis(14400)); // 4 hours, which is half the default wait_timeout of mysql
         config.setPoolName(db.name());
 
+        log.info("Created data source for {} with maxConnections={}, connectionTimeout={}s, maxLifetime={}s",
```

Comment:
When you create a log message you are free to add any (reasonable) text you'd like. Since log messages include the name of the class that produced them, that combined with the text of a given message is typically enough to find information in the logs.
What problem are you trying to solve?

Reply:
Just an idea. Reason is that I think a prefix makes it easier for searches. For example, if I want to search for all DB action logs (info, error), I'd type in `[DB]` and get just DB logs. Faster and precise.